facilitator/tests/integration_tests.rs

```diff
@@ -561,28 +560,28 @@ fn end_to_end_test(drop_nth_pha: Option<usize>, drop_nth_facilitator: Option<usi
 
     assert_eq!(aggregation_callback_count, 2);
 
-    let mut pha_aggregation_batch_reader: BatchReader<'_, SumPart, InvalidPacket> =
-        BatchReader::new(
-            Batch::new_sum(
-                instance_name,
-                &aggregation_name,
-                &start_date,
-                &end_date,
-                true,
-            ),
-            &mut *pha_aggregation_transport.transport,
-            false,
-            &TRACE_ID,
-            &logger,
-        );
-    let pha_sum_part = pha_aggregation_batch_reader.header(&pha_pub_keys).unwrap();
+    let pha_aggregation_batch_reader: BatchReader<'_, SumPart, InvalidPacket> = BatchReader::new(
```

Comment:
```suggestion
    let pha_aggregation_batch_reader = BatchReader::new(
```
Is the type annotation still needed?

Reply:
Whoops, nope, it is not. I missed cleaning it up because by `grep` invocation only searched `src/`, not `test/` 🙃.